synapse/models/person.py

```diff
@@ -57,6 +57,16 @@ def getModelDefs(self):
                 ('ps:vitals', ('guid', {}), {
                     'doc': 'Statistics and demographic data about a person or contact.',
                 }),
+                ('ps:skill', ('guid', {}), {
+                    'doc': 'A specific skill which a person or organization may have.'
+                }),
+                ('ps:skill:type:taxonomy', ('taxonomy', {}), {
```

Comment:
Do we want a form for `ps:skill:type:taxonomy` as well?

Reply:
Yep, good catch thanks!